app-vite/types/configuration/conf.d.ts

```diff
@@ -69,7 +69,7 @@ interface BaseQuasarConfiguration {
    */
   extras?: (QuasarIconSets | QuasarFonts)[];
   /** Add variables that you can use in index.template.html. */
-  htmlVariables?: { [index: string]: string };
+  htmlVariables?: Record<string, string | Record<string, unknown>>;
```

Comment:
Let's simply go with `any` as it's possible to use numbers, arrays, etc. as well.
```suggestion
  htmlVariables?: Record<string, any>;
```